src/pfe/file-watcher/scripts/nodejs-container.sh

```diff
@@ -246,6 +246,10 @@ function deployK8s() {
 	# Delete any pods left that are terminating, to ensure they go away
 	/file-watcher/scripts/kubeScripts/clear-terminating-pods.sh $project
 
+	POD_NAME="$( kubectl get po --selector=release=$project | grep 'Running' | cut -d ' ' -f 1 )"
```

Comment:
maybe put this logic in a function and use it to get the POD_NAME, there are a couple of places using the same logic..